source/daplink/interface/target_flash.c

```diff
@@ -65,15 +93,13 @@ static error_t target_flash_init()
         return ERROR_ALGO_DL;
     }
 
-    if (0 == swd_flash_syscall_exec(&flash->sys_call_s, flash->init, target_device.flash_start, 0, 0, 0)) {
-        return ERROR_INIT;
-    }
-
     return ERROR_SUCCESS;
 }
 
 static error_t target_flash_uninit(void)
 {
+    flash_func_start(FLASH_FUNC_NOP);
```

Comment:
A result is returned from `flash_func_start` but never checked. Is this intentional?

Reply:
Not intentional, will add return value check.